phpBB/phpbb/install/schemas/schema_data.sql

```diff
@@ -0,0 +1,821 @@
+#
```

Comment:
I mean these files are resources and I don't think we want to have any resources under phpBB/phpbb, I'm seeing this directory more like src/ or lib/ in some packages.